frontend/public/src/components/CourseProductDetailEnroll_test.js

```diff
@@ -664,55 +776,4 @@ describe("CourseProductDetailEnroll", () => {
 
     assert.isTrue(enrolledItem.exists())
   })
-
-  it("CourseInfoBox renders the archived message if the course is archived", async () => {
```

Comment:
It didn't, it just moved - there was a bunch of shuffling in this file because of the different renderers, figuring out how much I could convince enzyme to do. https://github.com/mitodl/mitxonline/pull/2030/files/9e4f6eccc567f15e6caa19c31837857555f462b0#diff-04610edacb3613ea80cd276232f83b4b3e771a18e7a25c3d2156e3e4d90ae9f8R363